src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -194,19 +194,14 @@ trait ClusterComponent extends LeoComponent {
     // find* and get* methods do query the INSTANCE table
 
     def getActiveClusterByName(project: GoogleProject, name: ClusterName): DBIO[Option[Cluster]] = {
-      fullClusterQuery
-        .filter { _._1.googleProject === project.value }
-        .filter { _._1.clusterName === name.value }
-        .filter { _._1.destroyedDate === Timestamp.from(dummyDate) }
+      fullClusterQueryByUniqueKey(project, name)
         .result map { recs =>
           unmarshalFullCluster(recs).headOption
         }
     }
 
     def getDeletingClusterByName(project: GoogleProject, name: ClusterName): DBIO[Option[Cluster]] = {
-      fullClusterQuery
-        .filter { _._1.googleProject === project.value }
-        .filter { _._1.clusterName === name.value }
+      fullClusterQueryByUniqueKey(project, name)
```

Comment:
still deciding if this is right to use here

Reply:
ok this appears to be correct. anything in `Deleting` state doesn't have a `destroyedDate` yet so it's okay if we use the default dummy date.